tarpc/src/macros.rs

```diff
@@ -569,6 +581,20 @@ mod functional_test {
         assert_eq!(3, client2.add(1, 2).get().unwrap());
     }
 
+    #[test]
+    fn async_try_clone_unix() {
+        let handle = Server.spawn_with_config(UnixTransport("/tmp/test"),
```

Comment:
This won't work if you run it twice because the file already exists. We can just delete it at the end I guess?/Choosing something like /tmp/tarpc-async_try_clone_UNIX?